library/build.gradle.kts

```diff
@@ -7,10 +7,23 @@ plugins {
     id("com.android.library")
     id("kotlinx-serialization")
     id("kotlin-android-extensions")
+    id("maven-publish")
+}
+
+group = "edu.usf.cutr.otp"
+version = "1.0.0"
+
+repositories {
```

Comment:
@BumbleFlash Was there a reason that a dependency on `mavenLocal()` was needed as part of this PR? Looking at this now (as part of #12), I don't think it's needed. Normally you'd have this in a project that was trying to use the library. You would run `./gradlew publishToMavenLocal` in this project, and then add the `mavenLocal()` in the application that depends on the library.

Reply:
Yeah, I don't think we need it as you said. I can remove it in the next PR.